### PR TITLE
Do not allow sending the ephemeral message to the empty conversation

### PR DESCRIPTION
**Pull request by mikeger**
# Issue

When sending the message to group conversation without the participants the sending is not possible, since there are no receivers (ephemeral is not sent to your own devices).

# Fix

Appending the message is failing in such conversations.

### CODE REVIEW

File: Tests/Source/Model/Conversation/ZMConversationTests+Ephemeral.swift [hunk 1]
```diff
@@ -84,6 +84,35 @@ class ZMConversationTests_Ephemeral : BaseZMMessageTests {
         // then
         XCTAssertEqual(conversation.messageDestructionTimeout, 5)
     }
+ 
+    func testThatSendingEphemeralToEmptyConversationIsNotPossible() {
+        // given
+        let conversation = ZMConversation.insertNewObject(in: uiMOC)
+        conversation.conversationType = .oneOnOne
+        conversation.conversationType = .group
```
Comment (typfel) [inline Tests/Source/Model/Conversation/ZMConversationTests+Ephemeral.swift]: Setting it twice? typo?

File: Tests/Source/Model/Conversation/ZMConversationTests+Ephemeral.swift [hunk 2]
```diff
@@ -84,6 +84,35 @@ class ZMConversationTests_Ephemeral : BaseZMMessageTests {
...
+                
+        // then
+        XCTAssertEqual(conversation.canSendEphemeral, false)
+    }
     
+    func testThatSendingEphemeralToNormalConversationIsPossible() {
+        // given
+        let conversation = ZMConversation.insertNewObject(in: uiMOC)
+        conversation.conversationType = .oneOnOne
+        conversation.conversationType = .group
```
Comment (typfel) [inline Tests/Source/Model/Conversation/ZMConversationTests+Ephemeral.swift]: same here

File: Tests/Source/Model/Conversation/ZMConversationTests+Ephemeral.swift (hunk 1 above)
Comment (mikeger) [inline Tests/Source/Model/Conversation/ZMConversationTests+Ephemeral.swift]: Yeah, my bad!

**Comment by mikeger** [comment]
@typfel please re-review

**Pull request merged**